website_anchor_smooth_scroll/__openerp__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antiun Ingeniería S.L. - Jairo Llopis
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Smooth Scroll for Website Anchors',
+    'summary': '''Replace default behavior for internal links (anchor only)
+     with smooth scroll
+    ''',
+    'version': '9.0.1.0.0',
+    'category': 'Website',
+    'website': 'https://laslabs.com/',
```

Comment:
That's a fair point. I initially tried to include both of them, but that was resulting in a broken link, so I settled for just linking to LasLabs. I'll revert back to the previous address.